e2e/setup/ingress-nginx.yaml

```diff
@@ -0,0 +1,628 @@
+apiVersion: v1
```

Comment:
Copied from 
https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml

In the kind ingress tutorial
https://kind.sigs.k8s.io/docs/user/ingress/#using-ingress